### ISSUE TITLE
Separate benchmarks/their statuses/selection scripts?

### ISSUE THREAD

**Issue by 5nizza** [SYNTCOMP/benchmarks#14]
Currently the benchmarks, their statuses, and the scripts to select benchmarks -- all are stored in the same repository.
As a result, each parameterized benchmark has a nested folder `parametric` [1]
there are also `csv` files that are used for (1) status reports and (2) benchmarks selection.
The item (1) maybe useful to users, but (2) does not seem so.
Further, if we decide not to use certain benchmarks in the competition, the only way to do this is to remove their parameters (_and statuses_) from the csv files, so that `genParametric.py` does not use them.

The suggestion is to separate the benchmarks from the their statuses and the selection scripts. It is not completely clear how to do this cleanly. Here are some thoughts.

Create one big csv file, with file names, and parameter values for parametric benchmarks. This file is fed to the benchmark-selection script. Then, create another csv file and store there all known statuses (and possibly model sizes, etc.). The drawback is the dependence on file names and folders structure (if someone renames/moves a file --> the csv file has to be updated as well).

**Comment by gaperez64** [comment]
I agree it's not optimal as is. My original idea was to add some sort of "inactive" flag in the individual csv files to have status report even for parameter values we don't generate benchmarks for but that is not very clean indeed. 

I like the idea of a separate big csv with benchmark names and parameter values to generate for! Not sure if I'll have time to implement this for this year though.
